tgui/packages/tgui/standby.js

```diff
@@ -0,0 +1,24 @@
+export const enterStandby = () => ({
+  type: 'standby',
+});
+
+export const standbyReducer = (state, action) => {
+  const { type } = action;
+
+  if (type === 'standby') {
+    return {
+      ...state,
+      data: {},
+      shared: {},
+      config: {
+        ...state.config,
+        title: '',
+        status: 1,
+        interface: '',
+      },
+      standby: true,
+    };
+  }
+
+  return state;
+};
```

Comment:
This would belong better to the backend module, since it's the part of the backend, and you DO access `standby` var via `useBackend()`, so it really makes sense to keep it all in one module.

Join reducers as well, and action type should also be called `backend/enterStandby`.